src/api/sendMessage.js

```diff
@@ -8,10 +10,14 @@ export default async (
   to: string | string[],
   subject: string,
   content: string,
+  local_id: number,
```

Comment:
Be consistent, either call it `localId` or rename `eventQueueId` to `queue_id` (I'd say the first is better)